python/pypio/pypio.py

```diff
@@ -0,0 +1,125 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+
+import atexit
+import json
+import os
+import sys
+
+from pypio.data import PEventStore
+from pypio.utils import dict_to_scalamap, list_to_dict
+from pypio.workflow import CleanupFunctions
+from pyspark.sql import SparkSession
+
+
+def init(app_name=None):
```

Comment:
While I'm not sure the use case of pypio, I feel the lack of consistency in these APIs.

For example, `find_events()` takes `app_name`. Does this mean that pypio supports multiple apps in a same session? If so, Can we call `init()` multiple times to create multiple apps?

I think either of following options is good for API consistency:

- If pypio supports multiple apps, separate a method for creating a new app from `init()`.
- If pypio doesn't support multiple apps, make `app_name` mandatory in `init()` and use it for `find_events()` too.

Reply:
You have a point. `init` function, as you say, shouldn't take `app_name`. I'll remove it.

Just an FYI, I found out the following:
- Running `pio app new <app_name>` creates an event table/index and an accesskey
- Running `pio accesskey new <app_name>` creates another accesskey for specified App
- Users must use an accessKey to import events via API

Reply:
Thanks for information! Actually, I had forgotten what `pio app new` (and `pio accessKey new`) do.

If you have a plan to support importing event data via Web API of the event server, how it handle accessKey can be an important point in the design of this Python support.

Reply:
I'm considering supporting data import, but I don't have an image of how to implement yet. The ideal API specification is the following:

```
diabetes_df = h2o.import_file("../input/diabetes.csv", destination_frame="diabetes_df")
```

I believe we should see the exact behavior of current commands and SDK. Anyway, this topic is out of scope of my PR.